tests/DurationTest.php

```diff
@@ -861,8 +863,8 @@ public function providerMultipliedBy() : array
             [3, 0, 3, 9, 0],
             [3, 1000, 3, 9, 3000],
             [3, 999999999, 3, 11, 999999997],
-            [1, 0,  \PHP_INT_MAX, \PHP_INT_MAX, 0],
-            [1, 0, \PHP_INT_MIN, \PHP_INT_MIN, 0],
+            [1, 0,  PHP_INT_MAX, PHP_INT_MAX, 0],
```

Comment:
Weird that the double-space has not been caught here?

Reply:
I'm interested in having this rule *most of the time*, but to be able to exclude it when done on purpose to improve readability, like in some of the tests. As you've done for other rules?

Reply:
@BenMorel Understood, however in this specific case the `DurationTest` file has column-aligned formatted arrays which shouldn't be touched so excluding the file is necessary and that will make this particular issue not detected and therefore not fixed. 

The only way to exclude the check for specific arrays in the same file would be to use a more fine grained approach using php comments

```php
// phpcs:ignore Squiz.Arrays.ArrayDeclaration.SingleLineNotAllowed
```

But I'm not sure the maintenance burden of applying these comments for every arrays is worth it. WDYT ?

For now I've excluded the whole `tests` folder from this `DuplicateSpacesSniff`

Reply:
I guess it's good enough for now, then!